scss/_patterns_table-expanding.scss

```diff
@@ -53,5 +55,9 @@
       }
     }
     // stylelint-enable selector-no-qualifying-type
+
+    .p-table-expanding__column-placeholder {
```

Comment:
I thought the reason for this change was to remove the dependency of hide utilities on the table pattern. As this is not the case due to the `u-hide-table-column` class being part of the hide utilities. Why not continue to use `u-hide` here for `display: none;`?

Reply:
This is a weird one. The expanding table is built in a way that requires one additional cell in every row (the hidden 'expandable' row). But, because it's additional `<td>` in a row, it needs to be accompanied by corresponding cell in header row for validity. So in the markup we have a `<td>` that is empty, not needed for the pattern itself (just for validity of markup) and has to be hidden.
Currently expanding table uses `u-hide` utility for it. But I don't think a pattern should depend on utility to provide a styling for its integral part. That's why I thought it would be better to have a pattern specific class name for this column placeholder (hence the name).

Personally I don't think the way expanding table was built is a good one from semantic and styling point of view, but it's not a place to change it completely. We can at least improve it to remove dependency on utility (that may or may not be available).

Reply:
As for the name, I don't know how to best name it. Is it an 'expanding column'? It doesn't seem to be a column at all - it may be in markup, but in the table we don't want it as column (we don't want it in header, and in rows we expand it below the row). So it's this weird implementation artifact that we want hidden from the view. I thought about just hiding the last cell in header row `.p-table-expanding thead td:last-child` but this feels quite fragile and much less explicit.

The benefit of `u-hide` here is that it's explicit what it does. But makes pattern dependent on a utility.
Maybe `p-table-expanding__hidden-cell` would be best of both words?

Reply:
We could target `&[aria-hidden='true'] {` and document that is how to hide the final header cell.

Reply:
Interesting idea. I like that better than inventing new class names, so will do that.